### DISCUSSION TITLE
HTML File formatting

### DISCUSSION THREAD

**Discussion by ryadios** [ItzDerock/discord-html-transcripts#144] [General]
In the previous versions we had `minify` property which made the html look nicer but in the latest version we don't so i was wondering if we could make the html formatting better in any way?
![image](https://github.com/ItzDerock/discord-html-transcripts/assets/96983394/d8108dc7-172d-480e-88a2-32860a9ae818)

**Comment by ItzDerock** [comment]
unfortunately the newer versions use react server side rendering, and there is no way to produce a non-minified output. You can run the output through something like [`html-prettify`](https://www.npmjs.com/package/html-prettify) if you want a cleaner output.

**Comment by ryadios** [comment]
Thank you for the response. I tried `html-prettify` and it didn't quite help so tried `prettier` formatting and it's working now
